modules/gitops-release-notes-1-14-4.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-4_{context}"]
+= Release notes for {gitops-title} 1.14.4
+
+{gitops-title} 1.14.4 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, 4.16, and 4.17.
```

Comment:
and 4.18?

Reply:
Hi @stevsmit, I have talked to QE @varshab1210 and confirmed with her that the highest OCP supported release version for this GitOps 1.14.4 is 4.17. The OCP 4.18 version is not supported for GitOps 1.14.4.